bot-run

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-source .env/bin/activate
-./run.sh >/dev/null 2>/dev/null &
+source /root/TLE/venv/bin/activate
```

Comment:
Better remove this file completely. Remove this file and commit is better i think.